tensorflow/python/kernel_tests/pooling_ops_test.py

```diff
@@ -45,11 +45,14 @@ def GetTestConfigs(include_nchw_vect_c=False):
     all the valid test configs as tuples of data_format and use_gpu.
   """
   test_configs = [("NHWC", False), ("NHWC", True)]
-  if not test.is_gpu_available(cuda_only=True):
+  if not test.is_gpu_available():
     tf_logging.info("NCHW and NCHW_VECT_C tests skipped because not run with "
-                    "--config=cuda or no GPUs available.")
+                    "--config=cuda or --config=sycl or no GPUs available.")
     return test_configs
   # "NCHW" format is currently supported exclusively on CUDA GPUs.
+  if "sycl" in test_util.gpu_device_name().lower():
```

Comment:
if you revert back change from line 48 ( if not test.is_gpu_available(cuda_only=True): ) that won't be needed